srcCpp03/perftest_publisher.cxx

```diff
@@ -33,8 +33,7 @@ RTI_UINT64 perftest_cpp::_Clock_usec = 0;
 const std::string perftest_cpp::_LatencyTopicName = "Latency";
 const std::string perftest_cpp::_AnnouncementTopicName = "Announcement";
 const std::string perftest_cpp::_ThroughputTopicName = "Throughput";
-const dds::core::Duration timeout_wait_for_ack =
-        dds::core::Duration(0, 10000000);
+const int timeout_wait_for_ack_nsec = 10000000;
```

Comment:
timeout_wait_for_ack_nsec should be unsigned long
timeout_wait_for_ack_sec should be long

Reply:
Done